Packer build appears to fail due to ansible non-zero exit

Packer build actually completes but appears to have failed:

```
2021/07/14 12:57:43 [INFO] (telemetry) ending ansible
==> qemu.openhpc-vm: Gracefully halting virtual machine...
2021/07/14 12:57:43 packer-builder-qemu plugin: Executing shutdown command: sudo shutdown -P now
2021/07/14 12:57:43 packer-builder-qemu plugin: [DEBUG] Opening new ssh session
2021/07/14 12:57:43 packer-builder-qemu plugin: [DEBUG] starting remote command: sudo shutdown -P now
==> qemu.openhpc-vm: Could not chdir to home directory /home/centos: No such file or directory
2021/07/14 12:57:43 packer-builder-qemu plugin: [ERROR] Remote command exited without exit status or exit signal.
2021/07/14 12:57:43 packer-builder-qemu plugin: Waiting max 5m0s for shutdown to complete
==> qemu.openhpc-vm: Converting hard drive...
2021/07/14 12:57:45 packer-builder-qemu plugin: VM shut down.
2021/07/14 12:57:45 packer-builder-qemu plugin: Executing qemu-img: []string{"convert", "-c", "-O", "qcow2", "/home/steveb/nrel-workshop-slurmapp/environments/nrel-workshop/images/ohpc-compute-210714-1229.qcow2", "/home/steveb/nrel-workshop-slurmapp/environments/nrel-workshop/images/ohpc-compute-210714-1229.qcow2.convert"}
2021/07/14 13:02:25 packer-builder-qemu plugin: stdout:
2021/07/14 13:02:25 packer-builder-qemu plugin: stderr:
==> qemu.openhpc-vm: Provisioning step had errors: Running the cleanup provisioner, if present...
2021/07/14 13:02:25 packer-builder-qemu plugin: failed to unlock port lockfile: close tcp 127.0.0.1:5958: use of closed network connection
2021/07/14 13:02:25 packer-builder-qemu plugin: failed to unlock port lockfile: close tcp 127.0.0.1:3406: use of closed network connection
Build 'qemu.openhpc-vm' errored after 32 minutes 59 seconds: Error executing Ansible: Non-zero exit status: exit status 2
```

Note that the `../environments/<environment>/images/*.qcow2` image has successfully been created.

This is presumably because ansible has moved centos's $HOME to /var/lib/centos during the play.

--- a/ansible/site.yml
+++ b/ansible/site.yml
@@ -28,4 +28,11 @@
   import_playbook: "{{ hook_path if hook_path | exists else 'noop.yml' }}"
   when: hook_path | exists
 
+- name: Shutdown Packer VM
+  hosts: builder
+  gather_facts: no
+  become: yes
+  tasks:
+    - community.general.shutdown:
+
 ...

--- a/packer/main.pkr.hcl
+++ b/packer/main.pkr.hcl
@@ -46,7 +46,6 @@ source "qemu" "openhpc-vm" {
     ["-cdrom", "config-drive.iso"]
     ]
   vm_name          = "ohpc-${var.groups[0]}-${local.timestamp}.qcow2" # image name
-  shutdown_command = "sudo shutdown -P now"
 }
 
 build {

